workspace/notebook/manage_main_pipeline_config_entry.json

```diff
@@ -0,0 +1,373 @@
+					"\"\"\"\n",
+					"\n",
+					"spark.sql(insert_sql)\n",
+					"print(f\"Entity '{entity_name}' inserted successfully with System_key = {new_entity_key}\")\n",
+					""
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"### Validate Insert"
+				]
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"collapsed": false
+				},
+				"source": [
+					"# Confirm the new entity exists\n",
+					"spark.sql(f\"\"\"\n",
+					"SELECT *\n",
+					"FROM odw_config_db.main_pipeline_config\n",
+					"WHERE entity_name = '{entity_name}'\n",
+					"\"\"\").show()\n",
+					""
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"### ⚠️Toggle Enablement (Once Function App is deployed)"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"# Set is_enable = true so the pipeline can pick it up\n",
+					"spark.sql(f\"\"\"\n",
+					"UPDATE odw_config_db.main_pipeline_config\n",
+					"SET is_enabled = true\n",
+					"WHERE entity_name = '{entity_name}'\n",
+					"\"\"\")"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"### Confirm the update"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"spark.sql(f\"\"\"\n",
+					"SELECT System_key, System_name, Object, entity_name, is_enabled\n",
+					"FROM odw_config_db.main_pipeline_config\n",
+					"WHERE entity_name = '{entity_name}'\n",
+					"\"\"\").show()"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"### List all entities in main_pipeline_config"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"spark.sql(\"\"\"\n",
+					"SELECT \n",
+					"    System_key, \n",
+					"    System_name, \n",
+					"    Object, \n",
+					"    entity_name, \n",
+					"    is_enabled\n",
+					"FROM odw_config_db.main_pipeline_config\n",
+					"ORDER BY System_key\n",
+					"\"\"\").show(n=100, truncate=False)\n",
+					""
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"### ⚠️Delete an Entity from main_pipeline_config"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"entity_to_delete = \"<your-entity-name-here>\"\n",
```

Comment:
By default this'll run since `bool(entity_to_delete)` is True. Maybe it's worth just setting it to `None` by default?

Reply:
No - There is a manual operation right after it which needs to be manually set to True